src/pages/login/Login.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Link, useLocation, useNavigate } from 'react-router-dom'
+// css 
+import './Login.css';
+// context
+import { useAuth } from 'context/authContext';
+
+function Login() {
+    const { updateLoginState } = useAuth();
+    const [userName, setUserName] = useState('')
+    const [password, setPassword] = useState('')
+    const navigate = useNavigate()
+    const location = useLocation()
+    const updateUserName = event => {
+        setUserName(event.target.value)
+    }
+    const updatePassword = event => {
+        setPassword(event.target.value)
+    }
+    const loginUser = (e) => {
+        e.preventDefault()
+        if(userName !== '' && password !== '') {
+            updateLoginState()
+            navigate(location?.state?.from?.pathname)
+        }
+    }
```

Comment:
Liked that you have taken care of empty values.
To add on you can also show alerts or error toasts for a better UX.

Reply:
Thanks for your review. Have added the deployment URL in the description, if you'd like to check. 

Would definitely add in next itreation
>To add on you can also show alerts or error toasts for a better UX.